chia/wallet/trade_manager.py

```diff
@@ -498,9 +500,13 @@ async def check_offer_validity(self, offer: Offer) -> bool:
         )
         return len(coin_states) == len(non_ephemeral_removals) and all([cs.spent_height is None for cs in coin_states])
 
-    async def calculate_tx_records_for_offer(self, offer: Offer, validate: bool) -> List[TransactionRecord]:
+    async def calculate_tx_records_for_offer(
+        self, offer: Offer, validate: bool, arbitrage_ph: Optional[bytes32] = None
+    ) -> List[TransactionRecord]:
         if validate:
-            final_spend_bundle: SpendBundle = offer.to_valid_spend()
+            final_spend_bundle: SpendBundle = offer.to_valid_spend(
+                arbitrage_ph
+            )  # arbitrage_ph is where the extra CAT's go.
```

Comment:
Can we remove this bit since we don't support arbitrage right now?

Reply:
yep!